pizzaserver-server/src/main/java/io/github/willqi/pizzaserver/server/network/protocol/packets/PlayerActionPacket.java

```diff
@@ -0,0 +1,85 @@
+package io.github.willqi.pizzaserver.server.network.protocol.packets;
+
+import io.github.willqi.pizzaserver.commons.utils.Vector3;
+import io.github.willqi.pizzaserver.server.Server;
+
+public class PlayerActionPacket extends BedrockPacket {
+
+    public static final int START_BREAK = 0;
```

Comment:
For these static ints, I'm not entirely sure if this is the best way to go about it. 
I know for the entity data packets the ids change from time to time as new properties get added. While the action packet hasn't been updated for quite some time, I feel that's it's better to be careful just in case they do add more actions in future versions. So maybe making this an enum would be better so that the PacketDeserializer can assign the type to it?